internal/middleware/middleware_test.go

```diff
@@ -659,7 +659,7 @@ func TestUserMgmtMiddleware_Cacher(t *testing.T) {
 				}
 				expected := &model.Response{
 					Status:  http.StatusBadRequest,
-					Message: "1004: unable to assert userid",
+					Message: "2004: unable to assert userid",
```

Comment:
Instead of hardcoding, use the codes.GetErr(codes.ErrAssertUserid) logic to make it dynamic to avoid rework when these are changed.

Reply:
done